hq/markdown/temporary-ssh.md

```diff
@@ -0,0 +1,29 @@
+# Temporary SSH
+
+In order to move towards use of temporary credentials, a facility has been provided to locally generate a keypair and push the
```

Comment:
This is very waffly. Attention is scarce, can we try and be as concise as possible in the docs?

Reply:
Done.